desktop/src/main/java/bisq/desktop/main/portfolio/openoffer/OpenOffersViewModel.java

```diff
@@ -118,6 +121,23 @@ String getMarketLabel(OpenOfferListItem item) {
         return CurrencyUtil.getCurrencyPair(item.getOffer().getCurrencyCode());
     }
 
+    String getPaymentMethod(OpenOfferListItem item) {
+        String result = "";
+        if (item != null) {
+            Offer offer = item.getOffer();
+            checkNotNull(offer);
+            checkNotNull(offer.getPaymentMethod());
+            String method = Res.get(offer.getPaymentMethod().getId() + "_SHORT");
```

Comment:
Can we guarantee that `Res.get(paymentMethod.getId() + "_SHORT")` will always find the translation? If unsure, should wrap in try-catch.

Reply:
It would be a good to move this duplicating logic to [`PaymentMethod`](https://github.com/bisq-network/bisq/blob/master/core/src/main/java/bisq/core/payment/payload/PaymentMethod.java) itself (`PaymentMethod.getShortName()` or something better). Put in a try-catch in that method for `java.util.MissingResourceException` (that's what `Res.get` might throw) and use an Option return type to distinguish whether or not the translation was found (again, since we can't guarantee (I presume) that someone won't forget to put in a translation when adding a new payment method).

Reply:
:+1: Done / refactored.  `Res.get` handles `MissingResourceException` itself and in that case returns the key.  I think that is a good way of handling the case of missing translations since it alleviates the caller from having to write custom error handling everywhere.

Reply:
@jmacxx I missed that, good catch.